src/pages/ShareYourToken/ShareYourToken.js

```diff
@@ -0,0 +1,42 @@
+import React, { useRef } from 'react';
+import { StyledForm, ReadOnlyInput } from './elements';
+import { Button } from '@material-ui/core';
+import { useSnackbar } from 'notistack';
+
+function ShareYourToken(props) {
+  let sharedUserToken = localStorage.getItem('userToken');
+  const { enqueueSnackbar } = useSnackbar();
+
+  const inputRef = useRef(null);
+  function copyData() {
+    navigator.clipboard.writeText(inputRef.current.value);
+    if (navigator.clipboard.writeText) {
+      enqueueSnackbar('Successfully copied!', {
+        variant: 'success',
+      });
+    }
+  }
+
+  return (
+    <>
+      <StyledForm>
+        <h2>Share your list</h2>
+        <p>Copy the token below to share your shopping list with others:</p>
+
+        <ReadOnlyInput value={sharedUserToken} readOnly={true} ref={inputRef} />
```

Comment:
You may only need to put `readOnly` in there (without the `={true}` part. Not 100% sure, but that's often how boolean props work.

Reply:
Yup, you're right! I don't need it. Changing it now...